install_config/default_imagestreams_templates.adoc

```diff
@@ -0,0 +1,420 @@
+endif::[]
+
+ifdef::openshift-enterprise[]
+If you also have the relevant xPaaS Middleware subscription active on your
+account, image streams and templates are also provided and supported by Red Hat
+for each of following middleware services:
+
+[horizontal]
+Middleware Services::
+- link:../using_images/xpaas_images/eap.html[JBoss EAP]
+- link:../using_images/xpaas_images/a_mq.html[JBoss A-MQ]
+- link:../using_images/xpaas_images/jws.html[JBoss Web Server]
+- link:../using_images/xpaas_images/fuse.html[JBoss Fuse Integration Services]
+- link:../using_images/xpaas_images/decision_server.html[Decision Server]
+- link:../using_images/xpaas_images/data_grid.html[JBoss Data Grid]
+endif::[]
+
+You can check which default image streams and templates are currently available
+in your OpenShift environment by logging into the web console and clicking *Add
+to Project*, or getting the lists for the *openshift* project using the CLI:
+
+----
+$ oc get is -n openshift
+$ oc get templates -n openshift
+----
+
+The JSON object definitions used to create these sets are installed on the file
+system of your master during
+ifdef::openshift-enterprise[]
+a link:../install_config/install/quick_install.html[quick] or
+link:../install_config/install/advanced_install.html[advanced installation].
+endif::[]
+ifdef::openshift-origin[]
+an link:../../install_config/install/advanced_install.html[advanced
+installation].
+endif::[]
+If you used another method to install, or if the default image streams and
+templates are ever removed or changed, you can use the following instructions to
+create the default objects yourself. Otherwise, the following instructions are
+not necessary.
+
+[[is-templates-prereqs]]
+== Prerequisites
+
+Before you can create the default image streams and templates (if they do not
+already exist):
+
+- The
+ifdef::openshift-enterprise[]
+link:../install_config/install/docker_registry.html[integrated Docker registry]
+endif::[]
+ifdef::openshift-origin[]
+link:../../install_config/install/docker_registry.html[integrated Docker
+registry]
+endif::[]
+service must be
+deployed in your OpenShift installation.
+- You must be able to run the following CLI commands with
+ifdef::openshift-enterprise[]
+link:../architecture/additional_concepts/authorization.html#roles[*cluster-admin*
+privileges],
+endif::[]
+ifdef::openshift-origin[]
+link:../../architecture/additional_concepts/authorization.html#roles[*cluster-admin*
+privileges],
+endif::[]
+because they operate on the default *openshift*
+ifdef::openshift-enterprise[]
+link:../architecture/core_concepts/projects_and_users.html#projects[project].
+endif::[]
+ifdef::openshift-origin[]
+link:../../architecture/core_concepts/projects_and_users.html#projects[project].
+endif::[]
+ifdef::openshift-origin[]
+- You must have cloned the
+https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_examples/files[repository]
+that contains the default image streams and templates:
```

Comment:
i thought the installer put them on the filesystem, why do they need to clone a repo to get them?
@sdodson